### PR TITLE
Update Algorithm Finder's Keepers

### PR DESCRIPTION
**Pull request by dakshshah96**
Refer #1094

### CODE REVIEW

**Comment by Rafase282** [comment]
@dakshshah96 this looks good, I would only ask that you add a link to our for loops article on the first solution.

**Comment by atjonathan** [comment]
### **LGTM :+1:**

**Comment by dakshshah96** [comment]
@Rafase282 I've added the link for For loops where required.

File: Algorithm-Finders-Keepers.md [hunk 1]
```diff
@@ -52,7 +61,18 @@ findElement([1, 2, 3, 4], function(num){ return num % 2 === 0; });
...
+### Code Explanation:
+
+- To make the code easier, create an undefined variable that will be returned.
+- Loop through the array to check for each element if it satisfy the function. This is done by passing the arr[index of the loop] as the parameter for the function from the second argument.
+- If true, then store the array element, and return it. This will stop the loop. No else needed.
+- If the loop was not broken and it has ended, then return **num** which by default is undefined. This means that none of the elements from the array satisfied the function.
+
+#### Relevant Links
+
+- [JS For Loops Explained](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-For-Loops-Explained)
```
Comment (atjonathan) [inline Algorithm-Finders-Keepers.md]: ~~[JS For Loops Explained](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-For-Loops-Explained)~~ [JS For Loops Explained](JS-For-Loops-Explained)

File: Algorithm-Finders-Keepers.md [hunk 2]
```diff
@@ -67,15 +87,21 @@ findElement([1, 2, 3, 4], function(num){ return num % 2 === 0; });
...
 :rocket: [Run Code](https://repl.it/CLn7/0)
 
-## Code Explanation:
+#### Relevant Links
 
-- To make the code easier, create an undefined variable that will be returned.
-- Loop through the array to check for each element if it satisfy the function. This is done by passing the arr[index of the loop] as the parameter for the function from the second argument.
-- If true, then store the array element, and return it. This will stop the loop. No else needed.
-- If the loop was not broken and it has ended, then return **num** which by default is undefined. This means that none of the elements from the array satisfied the function.
+- [JS Array Prototype Filter](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Filter)
```
Comment (atjonathan) [inline Algorithm-Finders-Keepers.md]: ~~`[JS Array Prototype Filter](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/JS-Array-Prototype-Filter)`~~ `[JS Array Prototype Filter](JS-Array-Prototype-Filter)`

File: Algorithm-Finders-Keepers.md (hunk 1 above)
Comment (dakshshah96) [inline Algorithm-Finders-Keepers.md]: Done

File: Algorithm-Finders-Keepers.md (hunk 2 above)
Comment (dakshshah96) [inline Algorithm-Finders-Keepers.md]: Done

**Comment by atjonathan** [comment]
#### **LGTM :+1:**
Thanks for contributing :tada:

**Pull request merged**